lib/index.js

```diff
@@ -112,7 +112,7 @@ function getBackgroundImageSize(decl, existingBackgroundSize, detectImageSize, r
           prop: 'background-size',
           value: `${size.width}px ${size.height}px`,
         });
-      });
+      }).catch(() => {});
```

Comment:
With `result.warn` - https://github.com/postcss/postcss/blob/master/docs/guidelines/plugin.md#32-use-resultwarn-for-warnings